app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
+    "description": "Tells the user what coin they have selected to deposit with shapeshift"
+  },
+  "depositEth": {
+    "message": "Depositar Eth"
+  },
+  "depositEther": {
+    "message": "Depositar Ether"
+  },
+  "depositFiat": {
+    "message": "Depositar con Fiat"
+  },
+  "depositFromAccount": {
+    "message": "Depositar con otra cuenta"
+  },
+  "depositShapeShift": {
+    "message": "Depositar con ShapeShift"
+  },
+  "depositShapeShiftExplainer": {
+    "message": "Si tu tienes otras cripto concurrencias, puedes tradear y depositar Ether directamente en tu wallet de Metamask. No necesitas cuenta"
+  },
+  "details": {
+    "message": "Detalles"
+  },
+  "directDeposit": {
+    "message": "Deposito directo"
+  },
+  "directDepositEther": {
+    "message": "Depositar directamente Ether"
+  },
+  "directDepositEtherExplainer": {
+    "message": "Si tu tienes algo de Ether, la forma rapida para tener Ether en tu nueva cartera es depositando directamente"
+  },
+  "done": {
+    "message": "Terminado"
+  },
+  "edit": {
+    "message": "Editar"
+  },
+  "editAccountName": {
+    "message": "Editar el nombre de la cuenta"
+  },
+  "encryptNewDen": {
+    "message": "Encriptar tu nuevo DEN"
+  },
+  "enterPassword": {
+    "message": "Ingresa contraseña"
+  },
+  "passwordCorrect": {
+    "message": "Asegurate que tu contraseña es correcta"
+  },
+  "etherscanView": {
+    "message": "Ver la cuenta en Etherscan"
+  },
+  "exchangeRate": {
+    "message": "Tipo de cambio"
+  },
+  "exportPrivateKey": {
+    "message": "Exportar llave privada"
+  },
+  "exportPrivateKeyWarning": {
+    "message": "Exportar llaves privadas bajo tu propio riesgo."
+  },
+  "failed": {
+    "message": "Fallo"
+  },
+  "fiat": {
+    "message": "FIAT",
+    "description": "Exchange type"
+  },
+  "fileImportFail": {
+    "message": "Importar archivo no funciona? Click Aqui!",
+    "description": "Helps user import their account from a JSON file"
+  },
+  "from": {
+    "message": "De:"
+  },
+  "fromShapeShift": {
+    "message": "De ShapeShift"
+  },
+  "gas": {
+    "message": "Gas",
+    "description": "Short indication of gas cost"
+  },
+  "gasFee": {
+    "message": "Comision de gas"
+  },
+  "gasLimit": {
+    "message": "Limit de gas"
+  },
+  "gasLimitCalculation": {
+    "message": "Calculamos el límite de gas sugerido en función de las tasas de éxito de la red."
+  },
+  "gasLimitRequired": {
+    "message": "Limite de Gas requerido"
+  },
+  "gasLimitTooLow": {
+    "message": "El límite de gas debe ser de al menos 21000"
+  },
+  "gasPrice": {
+    "message": "Precio del Gas (GWEI)"
+  },
+  "gasPriceCalculation": {
+    "message": "Calculamos los precios sugeridos del gas en función de las tasas de éxito de la red."
+  },
+  "gasPriceRequired": {
+    "message": "Precio del GAS requerido"
+  },
+  "getEther": {
+    "message": "Conseguir Ether"
+  },
+  "getEtherFromFaucet": {
+    "message": "Obtenga Ether desde un foucet por $1",
```

Comment:
`Obtenga Ether de un faucet (grifo) por $1` @danfinlay confirm: Are we giving ether for USD$ 1? How does it work?